src/utils/frontend/FormSubmissionHelper.js

```diff
@@ -12,14 +12,16 @@ import { uploadDocument, updateDocument } from '../network/session-core';
  * @memberof utils
  * @function makeHandleFormSubmission
  * @param {string} uploadType - Type of upload (cross, self, etc.) to perform
+ * @param {string} expireDate - document expiration date
```

Comment:
I believe you're using the object from Day.js for expireDate. If that's the case the type for this should be Dayjs not string.

You would need to import the typedef first like with Session from Inrupt's library. So something like this:

```
/**
 * @typedef {import("dayjs").Dayjs} Dayjs
 */
```